### DISCUSSION TITLE
can one use azure object storage

### DISCUSSION THREAD

**Discussion by tshepang** [quickwit-oss/quickwit#1873] [Q&A] [answered]
One can use S3 as primary storage with Quickwit, so I do wonder if one can similarly use Azure object storage.

**Comment by fmassot** [comment, marked as answer]
Hi @tshepang 

We added recently support for Azure storage, but we did not make a release since that, so you need to compile yourself quickwit with `--features azure` if you want to use it.

You have some docs here: https://github.com/quickwit-oss/quickwit/blob/main/docs/guides/azure-setup.md
